docs/modules/user-guide/partials/proc_migrating-components.adoc

```diff
@@ -2,13 +2,11 @@
 = Migrating components
 
 [role="_abstract"]
-This section describes how to migrate components to devfile v2. There ARE major changes in the `components` section of the devfile. A component specified in a v1.0 devfile will NOT work as it is in a devfile v2.0.
+This section describes how to migrate components to devfiles v2.x. The components section of devfiles v2.x has major changes. A component specified in a devfile v1.x no longer works in a devfile v2.x.
 
 .Procedure
 
-. Component is a polymophic type
-+
-For a better syntax validation component now is defined as a polymorphic type and can be implemented as `container`, `kubernetes`, `openshift`, `plugin` or `volume`.
+. To get a better syntax validation, define a component as a polymorphic type and implement it as a `container`, `kubernetes`, `openshift`, `plug-in` or `volume`.
```

Comment:
"a `container`, `kubernetes`, `openshift`, `plug-in` or `volume`."
it looks like container has an article, but the rest doesn't. Maybe that's what it's supposed to be like, but plug-in or volume should have an article, too, right? Is it possible to say "a kubernetes" or "an openshift"? If so, disregard my comment, please. If not, maybe
"a `container`, `plug-in`, `volume`, or `kubernetes` or `openshift`" could be better? Not sure about this.

Reply:
Or delete the article? Really not sure.